model/buildlogger.go

```diff
@@ -511,7 +532,9 @@ func createFindQuery(opts LogFindOptions) map[string]interface{} {
 	if opts.Info.Format != "" {
 		search[bsonutil.GetDottedKeyName(logInfoKey, logInfoFormatKey)] = opts.Info.Format
 	}
-	if len(opts.Info.Tags) > 0 {
+	if opts.Group != "" {
+		search[bsonutil.GetDottedKeyName(logInfoKey, logInfoTagsKey)] = bson.M{"$in": []string{opts.Group}}
```

Comment:
Isn't is sufficient to omit `$in` for the case of a singular string? I think `search[bsonutil.GetDottedKeyName(logInfoKey, logInfoTagsKey)] = opts.Group` is equivalent.